report-viewer/src/components/ComparisonsTable.vue

```diff
@@ -247,32 +359,50 @@ export default defineComponent({
 </script>
 
 <style scoped>
-table {
+.table {
+  table-layout: fixed;
   border-collapse: collapse;
   font-size: larger;
   text-align: center;
 }
 
-tr:nth-child(odd) {
-  background: var(--primary-color-light);
-}
-
-tr:nth-child(even) {
-  background: var(--secondary-color);
-}
-
 th {
   margin: 0;
   padding-top: 2%;
   padding-bottom: 2%;
   color: var(--on-primary-color);
 }
 
-td {
+.td1 {
+  padding-right: 10%;
   padding-top: 3%;
   padding-bottom: 3%;
 }
 
+.td2 {
```

Comment:
Are these different `td` classes really required? Also, they break the view if the content gets to long, see my example in the comment after the review

Reply:
td1...td6: i need these classes to control them, because each comparison is a table. It can't be aligned automatically, because new component brokes the table structure.